mne/report.py

```diff
@@ -765,7 +765,7 @@ def _add_figs_to_section(self, figs, captions, section='custom',
             # on some version mayavi.core won't be exposed unless ...
             from mayavi import mlab  # noqa, mlab imported
             import mayavi
-        except ImportError:
+        except:  # on some systems importing Mayavi raises SystemExit (!)
```

Comment:
I'm much less interested in learning the various failure modes of a
different package than I am in ensuring our package doesn't throw
unexpected and cryptic errors like this was doing...